### ISSUE TITLE
Portuguese (pt-br) Translation

### ISSUE THREAD

**Issue by joaopgrassi** [nbarbettini/little-aspnetcore-book#56]
Hi!
This seems to be a good hit in Brazil. There's no much content in Portuguese for ASP.NET Core and (believe me) there's bunch of devs in Brazil who can't read in English.

I could help translating it.. just let me know if you are interested.. and we could proceed from that. 

Cheers.

**Comment by nbarbettini** [comment]
I would love to have a Portuguese translation! I just released a new version (1.1) that includes a lot of copy changes so now is a great time to perform the translation. Let me know how I can help.

I'm curious, do you know how folks in Brazil finding the book?

**Comment by joaopgrassi** [comment]
Cool! I have some limited time, but I'm definitely excited in working on this! 

Since this is new to me, maybe you can fill me in on how was the coordination for the other translations? I was thinking on creating a pt-br branch where I would work. Of course, keep merging in master or whatever branch has new things.. until it's released? 

Hum.. probably through just googling, but in Portuguese! I know bunch of people devs that still Google stuff in Portuguese.. so if we manage to 'disseminate' the contents of the book in Portuguese, via blogs or other channels... people will find it in the end. I have a blog, so I can definitely write a post and share with my network, which is not much but.. it's a start :).

**Comment by joaopgrassi** [comment]
I liked your idea on #35, could pursue like that. Not sure about the Gitbook part though. Do you want to keep them all in the same project, or should I host it on my personal account like the others did?

**Comment by nbarbettini** [comment]
The gitbook stuff is a little weird, but we can figure that out later. I think a fork + branch is the best approach, like this one: https://github.com/windsting/little-aspnetcore-book (see branch `master` and `zh-cn`).

The hard/time-consuming part is translating it. I can help with publishing and gitbook stuff when it's ready!

**Comment by joaopgrassi** [comment]
Yeah, I just spent 1 hour on Gitbook. It's a mess. I did the integration on the new Gitbook, since cannot create account in the legacy anymore. I managed to import my pt-br branch into it.. but it did lot's of changes. 
Just gonna give up for now on setting up Gitbook, and will only focus on translating it.

**Comment by nbarbettini** [comment]
Ugh, sorry about that. If you do the translation in a fork, then we can figure all that stuff out later. 👍

**Comment by joaopgrassi** [comment]
No worries. Will keep this updated :).

**Comment by nicolastarzia** [comment]
@joaopgrassi, do you need help to translate this book?
I can help you!

**Comment by nicolastarzia** [comment]
Yes, I think splitting by chapter.
Can we use Project board in your project? 

Just to see all pending chapters...

**Comment by joaopgrassi** [comment]
That's a good idea. I'll add a project board.. and you as contributor. Since I didn't started anything.. you can pick any chapter.. but first I would need to create all tasks on the board.. maybe I can do it this weekend. We'll have to find a chat so we don't spam everyone here. Skype's cool for you?

**Comment by joaopgrassi** [comment]
Ok so @nicolastarzia I added a project for the translation.. and also created issues so we assign to ourselves and have a nice track of who's doing what. Thanks for the idea! Also, I need you guys specially @nicolastarzia  to check this: https://github.com/joaopgrassi/little-aspnetcore-book/issues/11. I need your opinion on the Title of the book in Portuguese.

**Comment by joao-marcos** [comment]
Hi guys
I'm looking for projects that need translation. If I can, please let me know. I am not an expert, but I believe that with your supervision everything will work out.

**Comment by nbarbettini** [comment]
@joao-marcos See @joaopgrassi's fork here: https://github.com/joaopgrassi/little-aspnetcore-book/issues
He's got issues defined for translating each chapter of the book. Check with him to see if he needs help!

**Comment by joao-marcos** [comment]
Nice! @joaopgrassi if theres no problem, I can take the chapter "MVC basics", ok?

**Comment by joaopgrassi** [comment]
@joao-marcos any help is welcome :). Please, fork my fork of the book, and make sure you branch out of https://github.com/joaopgrassi/little-aspnetcore-book/tree/pt-br branch. I'm using a pattern for the branch names, of chapter/{chapter_name}. See the one I'm working on for an example.
Also, make sure you assign yourself the issue and update it accordingly in the project board. 
When you are ready, create a pull request so we can review it. 
Thanks.

**Comment by joaopgrassi** [comment]
Also, I think it's a good idea to merge the pt-br branch into your branch to avoid merge conflicts. This way it's always a fast-forward.

**Comment by joao-marcos** [comment]
All right! Thank you

**Comment by oliveiramarcio** [comment]
I forked your fork and I will start to translate "add-external-packages" chapter first.

**Comment by joaopgrassi** [comment]
@oliveiramarcio see my previous comment for @joao-marcos on how to get started. I will add you as a contributor.. so you assign the chapter ticket to yourself and move the tickets on the project. But before you start, I would assign you to review the chapter that I translated and it's pending review. Can you do that?

**Comment by joaopgrassi** [comment]
Also @oliveiramarcio that chapter is already picked by @nicolastarzia. Always check the board to see who's doing what. https://github.com/joaopgrassi/little-aspnetcore-book/projects/1

**Comment by oliveiramarcio** [comment]
Hi @joaopgrassi!

I'll review the _"introduction"_ chapter and report you soon.

The after the review I'll start translating _"add-external-packages"_ chapter.

**Comment by joaopgrassi** [comment]
Cool. I've sent you a collaborator invite. Once you accept, you can assign the ticket to you and move it on the board. Thanks!

**Comment by joaopgrassi** [comment]
Hey all. It's been a while since we started on the pt-BR translations. As always, things happen and a lot has changed since I started with this project. Unfortunately, I don't think I can contribute more as my time is limited and I'm swamped with other stuff atm. To not lose all the hard work we've done, I
transferred the fork to @nicolastarzia, so you guys can continue working on this, and hopefully finishing it :)

I would like to thank you all that helped to translate it so far, and I'm sorry for not having been much active on it.

**Comment by nicolastarzia** [comment]
Thank you @joaopgrassi, you made a good job. 
When you have some time, would be awesome that you can translate with us! :v:

**Comment by LuizFurtado** [comment]
Hi Guys!

Is this translation work still on going? Would you be willing to accept an extra pair of hands on the work? I had never worked on such project before, but I would love to learn to help on the job to translate it to Brazilian portuguese, as I understood this book would be of great help to some non-english enthusiasts.
